demo/menu-bar-items-demos.html

```diff
@@ -0,0 +1,138 @@
+<dom-module id="menu-bar-items-demos">
+  <template>
+    <style include="vaadin-component-demo-shared-styles">
+      :host {
+        display: block;
+      }
+    </style>
+
+    <h3>Disabled Items</h3>
```

Comment:
We don't have a demo of changing the disabled-state dynamically. I think that might be a common use-case, and it would be good to demonstrate because it needs a call to `render`, which is not obvious, eg.
```js
menu[1].disabled = false;
menu.render();
```

Reply:
Thanks, done.